code/modules/power/solar.dm

```diff
@@ -21,21 +20,20 @@
 	var/obj/machinery/power/solar_control/control
 	var/needs_to_turn = TRUE //do we need to turn next tick?
 	var/needs_to_update_solar_exposure = TRUE //do we need to call update_solar_exposure() next tick?
-	var/obj/effect/overlay/panel
+	var/mutable_appearance/panelstructure
+	var/mutable_appearance/paneloverlay
+	var/multiplier
+	var/panelcolor
 
```

Comment:
```suggestion
	var/obj/item/stack/sheet/glass_type

```

Reply:
this doesnt work.

Reply:
> this doesnt work.

oh really

Reply:
> > this doesnt work.
> 
> oh really

byond quirk where initial makes it valid for not a very logical reason